.gitignore

```diff
@@ -48,3 +48,5 @@ web-build/
 # Cypress
 /cypress/screenshots
 /cypress/videos
+
+/cypress-visual-screenshots/**
```

Comment:
the jpg to check against need to be committed

Reply:
@fuxingloh the QR code varies per run. It creates the snapshot first -> exit wallet -> load and check the snapshot

Reply:
Oh, I think we should have a fixed entered mnemonic and check against the QR code to detect regression in QR generation.